src/json.hpp

```diff
@@ -10929,7 +10955,7 @@ class basic_json
 
     @since version 1.0.0
     */
-    size_type count(typename object_t::key_type key) const
+    size_type count(object_comparator_key_t key) const
```

Comment:
Would the `find` and `count` functions be better as templates with perfect forwarding, so we can find or count directly anything that the underlying object class can find or count?